wandb_workspaces/reports/v2/interface.py

```diff
@@ -1563,41 +1962,60 @@ def from_model(cls, model: internal.ScatterPlot):
             custom_expressions=model.config.expressions,
             legend_template=model.config.legend_template,
             font_size=model.config.font_size,
-            layout=Layout.from_model(model.layout),
+            layout=Layout._from_model(model.layout),
         )
         obj._id = model.id
         return obj
 
 
 @dataclass(config=dataclass_config, repr=False)
 class CodeComparer(Panel):
+    """
+    A panel object that compares the code between two different runs.
+
+    Attributes:
+        diff (Required): How to display code differences.
```

Comment:
consistency: Maybe worth going over whether you want to include:
1. Types
2. "Required"

And then apply that formatting standard to all objs